### PR TITLE
tweak(clrcore): add new bones

### PR DESCRIPTION
**Pull request by koketoo**
Added the bone ids that weren't included

### Goal of this PR

Include the bond ids that weren't included
...

### How is this PR achieving the goal
Adding the bone ids
...

### This PR applies to the following area(s)

FiveM
...

### Successfully tested on

**Game builds:** .. 3095

**Platforms:** Windows

### Checklist

- [ ] Code compiles and has been tested successfully.
- [ ] Code explains itself well and/or is documented.
- [ ] My commit message explains what the changes do and what they are for.
- [ ] No extra compilation warnings are added by these changes.

### CODE REVIEW

File: code/client/clrcore/External/BoneID.cs [hunk 1]
```diff
@@ -27,27 +27,27 @@ public enum Bone
...
-		SKEL_L_Finger22,
+		SKEL_L_Finger22 = 64113,
 		SKEL_L_Finger30 = 26613,
 		SKEL_L_Finger31 = 4137,
-		SKEL_L_Finger32,
+		SKEL_L_Finger32 = 64065,
 		SKEL_L_Finger40 = 26614,
 		SKEL_L_Finger41 = 4153,
-		SKEL_L_Finger42,
+		SKEL_L_Finger42 = 64081,
```
Comment (PsychoShock) [inline code/client/clrcore/External/BoneID.cs:50]:
```suggestion
		SKEL_L_Finger42 = 4154,
```

The `64081` is for the right finger42 :)

File: code/client/clrcore/External/BoneID.cs [hunk 2]
```diff
@@ -27,27 +27,27 @@ public enum Bone
...
+		SKEL_Spine2 = 24817,
+		SKEL_Spine3 = 24818,
 		SKEL_L_Clavicle = 64729,
 		SKEL_L_UpperArm = 45509,
 		SKEL_L_Forearm = 61163,
 		SKEL_L_Hand = 18905,
 		SKEL_L_Finger00 = 26610,
 		SKEL_L_Finger01 = 4089,
-		SKEL_L_Finger02,
+		SKEL_L_Finger02 = 64017,
```
Comment (PsychoShock) [inline code/client/clrcore/External/BoneID.cs:38]:
```suggestion
		SKEL_L_Finger02 = 4090,
```
The `64017` is for the right finger02 :)

File: code/client/clrcore/External/BoneID.cs [hunk 3]
```diff
@@ -27,27 +27,27 @@ public enum Bone
...
 		SKEL_L_Finger10 = 26611,
 		SKEL_L_Finger11 = 4169,
-		SKEL_L_Finger12,
+		SKEL_L_Finger12 = 64097,
```
Comment (PsychoShock) [inline code/client/clrcore/External/BoneID.cs:41]:
```suggestion
		SKEL_L_Finger12 = 4170,
```

The `64097` is for the right finger12 :)

File: code/client/clrcore/External/BoneID.cs [hunk 4]
```diff
@@ -27,27 +27,27 @@ public enum Bone
...
 		SKEL_L_Finger20 = 26612,
 		SKEL_L_Finger21 = 4185,
-		SKEL_L_Finger22,
+		SKEL_L_Finger22 = 64113,
```
Comment (PsychoShock) [inline code/client/clrcore/External/BoneID.cs:44]:
```suggestion
		SKEL_L_Finger22 = 4186,
```

The `64113` is for the right finger22 :)

File: code/client/clrcore/External/BoneID.cs [hunk 5]
```diff
@@ -27,27 +27,27 @@ public enum Bone
...
 		SKEL_L_Finger30 = 26613,
 		SKEL_L_Finger31 = 4137,
-		SKEL_L_Finger32,
+		SKEL_L_Finger32 = 64065,
```
Comment (PsychoShock) [inline code/client/clrcore/External/BoneID.cs:47]:
```suggestion
		SKEL_L_Finger32 = 4138,
```

The `64065` is for the right finger32 :)

**Review by PsychoShock — COMMENTED** [review]
Hello,

Small mistakes, I did some suggestions with the right bone id for you :)

Nice job.

**Comment by koketoo** [comment]
> Hello,
> 
> Small mistakes, I did some suggestions with the right bone id for you :)
> 
> Nice job.

Done!

**Review by FabianTerhorst — CHANGES_REQUESTED** [review]
Can you update your commit message to fit our commit style?

```
tweak(clrcore): add new bones
```

**Comment by koketoo** [comment]
> Can you update your commit message to fit our commit style?
> 
> ```
> tweak(clrcore): add new bones
> ```

I'm starting to get desperate, the title is still wrong.

**Comment by koketoo** [comment]
@FabianTerhorst Okey, let me clone the repository on GitHub Desktop.

**Comment by koketoo** [comment]
@FabianTerhorst Could you please do it? I'm getting error on GitHub desktop.

**Comment by FabianTerhorst** [comment]
> @FabianTerhorst Could you please do it? I'm getting error on GitHub desktop.

You can also try to apply the changes again on a new client clone. GitHub desktop is not required for that.

**Comment by FabianTerhorst** [comment]
Will manually pull the changes, change the commit and remove the line format change.

**Comment by FabianTerhorst** [comment]
Merged https://github.com/citizenfx/fivem/commit/14ddd6a9fe6ca0c61d5a29776be505e5e9eed20c

**Pull request closed without merging**